src/main/java/dev/j3fftw/litexpansion/Items.java

```diff
@@ -272,6 +273,42 @@
         LoreBuilderDynamic.powerPerTick(RubberSynthesizer.ENERGY_CONSUMPTION)
     );
     //// Solar panels
+    public static final SlimefunItemStack BASIC_SOLAR_PANEL = new SlimefunItemStack(
+        "BASIC_SOLAR_PANEL",
+        Material.BLUE_GLAZED_TERRACOTTA,
+        "&3&lBasic Solar Panel",
+        "&6Only works during the day",
```

Comment:
no need to state that it works during the day since
euhm 
its a solar panel
same for the others